rpcs3/Emu/Cell/Modules/cellMsgDialog.h

```diff
@@ -54,6 +54,17 @@ enum : s32
 	CELL_MSGDIALOG_BUTTON_ESCAPE  = 3,
 };
 
+// Only for use with the cellMsgDialogOpen function
+enum : u32
+{
+	CELL_MSGDIALOG_DIALOG_TYPE_ERROR = 0 << 0,
```

Comment:
can you also indent these as seen in the  enum above?
Makes it easier to read and also consistent in style.